### PR TITLE
Update timespan form when structure changes

### PR DESCRIPTION
**Pull request by Dananji**

### CODE REVIEW

File: src/components/TimespanForm.js [hunk 1]
```diff
@@ -51,6 +51,9 @@ class TimespanForm extends Component {
   }
 
   componentWillReceiveProps(nextProps) {
```
Comment (adamjarling) [inline src/components/TimespanForm.js]: @Dananji I think`componentWillReceiveProps` may be a deprecated lifecycle method.  Can you see if `getDerivedStateFromProps` or some other method can give you the timing you need on this?

Comment (Dananji) [reply src/components/TimespanForm.js]: @adamjarling  Yeah, `componentWillReceiveProps` is going to be removed in version 17. I will see there's any other way to do this including `getDerivedStateFromProps`. Should I create a different ticket for this one? I feel like we might need to change some stuff around for this.

Comment (adamjarling) [reply src/components/TimespanForm.js]: Yea, if we have the time now to fix, may as well get ahead of it?  I'd go ahead and create the ticket.

Comment (Dananji) [reply src/components/TimespanForm.js]: Created issue to fix this, #16

**Pull request closed without merging**